src/test/java/uk/gov/companieshouse/api/accounts/service/impl/StatementServiceTest.java

```diff
@@ -115,12 +121,17 @@ void shouldCreateStatement() throws DataException {
         when(transactionMock.getLinks()).thenReturn(transactionLinksMock);
         when(transactionLinksMock.getSelf()).thenReturn(SELF_LINK);
 
+        ResponseObject responseObject = new ResponseObject(ResponseStatus.FOUND);
+        when(profitAndLossService.find(COMPANY_ACCOUNTS_ID, uk.gov.companieshouse.api.accounts.enumeration.AccountingPeriod.CURRENT_PERIOD)).
```

Comment:
Do we need the fully qualified package name here?

Reply:
Yes because we already have an AccountingPeriod imported from model

Reply:
👍